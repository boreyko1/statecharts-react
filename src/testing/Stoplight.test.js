import { testStatechart } from 'react-automata';
import { statechart, StoplightComponent } from './Stoplight';

test('Stoplight', () => {
  testStatechart({ statechart }, StoplightComponent);
});
