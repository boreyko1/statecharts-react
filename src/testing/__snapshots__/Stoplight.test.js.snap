// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stoplight: green 1`] = `
<div>
  Green
  <button
    onClick={[Function]}
    type="button"
  >
    Next Light
  </button>
</div>
`;

exports[`Stoplight: red 1`] = `
<div>
  Red
  <button
    onClick={[Function]}
    type="button"
  >
    Next Light
  </button>
</div>
`;

exports[`Stoplight: yellow 1`] = `
<div>
  Yellow
  <button
    onClick={[Function]}
    type="button"
  >
    Next Light
  </button>
</div>
`;
